### ISSUE TITLE
[Adaptive Cards> Close button]: Tooltip is not defined for the 'Close' button which is present on left pane navigation controls.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9238]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'Close' button present on left pane navigation.
3. Observe the issue.

**Actual Result:**
For 'Close' button on the left pane navigation, tooltip is not defined.

**Expected Result:**
For 'Close' button, proper and dismissible (Esc Key) tooltip should be defined, or shortcut key should be defined for the tooltip such as 'Ctrl+ Shift+ F10 to access the tooltip.

**User Impact:**
Person with cognitive disability will get impacted if tooltip is not defined due to which user will not be able to interact with the content or will have difficulty in understanding the purpose of the content properly.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/fd766a37-f1ea-4cc6-8ab1-1bf99ee2a432)

https://github.com/user-attachments/assets/26d76aad-16f9-4b8e-82df-accc397a23f3

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug ID: #7786
